<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="12492" started-at="2014-10-26T20:03:21Z" finished-at="2014-10-26T20:03:33Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="12492" started-at="2014-10-26T20:03:21Z" finished-at="2014-10-26T20:03:33Z">
      <class name="com.hd.framework.MainTestNGClass">
        <test-method status="FAIL" signature="beforeTest()[pri:0, instance:com.hd.framework.MainTestNGClass@10d6c27]" name="beforeTest" is-config="true" duration-ms="0" started-at="2014-10-26T20:03:21Z" finished-at="2014-10-26T20:03:21Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.hd.framework.MainTestNGClass.beforeTest(MainTestNGClass.java:33)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.TestRunner.beforeRun(TestRunner.java:641)
	at org.testng.TestRunner.run(TestRunner.java:609)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="SKIP" signature="f()[pri:0, instance:com.hd.framework.MainTestNGClass@10d6c27]" name="f" duration-ms="16" started-at="2014-10-26T20:03:33Z" finished-at="2014-10-26T20:03:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- f -->
        <test-method status="SKIP" signature="afterTest()[pri:0, instance:com.hd.framework.MainTestNGClass@10d6c27]" name="afterTest" is-config="true" duration-ms="0" started-at="2014-10-26T20:03:33Z" finished-at="2014-10-26T20:03:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- com.hd.framework.MainTestNGClass -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
